src/snowflake/snowpark/dataframe_writer.py

```diff
@@ -246,6 +268,12 @@ def save_as_table(
                 max_data_extension_time,
                 change_tracking,
                 copy_grants,
+                is_iceberg,
```

Comment:
for iceberg only params, can we raise a value error if those params are provided when iceberg table is not set to true?